doc/user/routemap.rst

```diff
@@ -277,6 +277,11 @@ Route Map Set Command
 
    Set the BGP-4+ link local IPv6 nexthop address.
 
+.. index:: set origin ORIGIN
+.. clicmd:: set origin ORIGIN
+
+   Set BGP route origin. Options are 1) igp, 2) egp, or 3) incomplete.
```

Comment:
Remove the second sentence and make the documented command above

`set origin ORIGIN <egp|igp|incomplete>` and I'll merge this

Reply:
Done.